File: tests/fixtures/python/slop_restate_pt_br.py
class HitCounter:
    def __init__(self):
        self.contador = 0

    def record_hit(self):
        # incrementa o contador
        self.contador += 1

    def record_batch(self, n):
        self.contador += n  # incrementa o contador


# expect-line: 6 SLOP042
# expect-line: 10 SLOP042

File: tests/fixtures/typescript/slop_restate_pt_br.ts
class HitCounter {
  contador = 0;

  registrarHit(): void {
    // incrementa o contador
    this.contador += 1;
  }

  registrarLote(n: number): void {
    this.contador += n; // incrementa o contador
  }
}

// expect-line: 5 SLOP042
// expect-line: 10 SLOP042
